### ISSUE TITLE
Brightness control by software ( MPI7002 )

### ISSUE THREAD

**Issue by lukecanny** [goodtft/LCD-show#282]
Is there an interface to interact with the MPI7002 7" Touchscreen backlight through software? I read on this page that there is no hardware to realise the software control for another model touchscreen.
Thank you.

**Comment by almirus** [comment]
do you found any solution?

**Comment by DJTommek** [comment]
Maybe it is offtopic, but for some it might be useful.

My usage of display is as HTML page so I came up with this easy workaround using CSS and Javascript by creating black overlay over page with initial opacity 0 (which is invisible). Then you are able to control brightness via changing opacity from 0 (100% brightness) to 0.8 (20% brightness), via button, websocket, AJAX, etc.

I've created simple demo on Fiddle, https://jsfiddle.net/DJTommek/ovugep7r/ including persisting brightness after page refresh.
